### DISCUSSION TITLE
Batchlink Substitute

### DISCUSSION THREAD

**Discussion by Ayaya70** [NUROISEA/anime-webui-colab#51] [Ideas]
Hey, I found this alternative to Batchlink (since it’s not working anymore) called HUB. If it’s not too much trouble, it would be great to add this extension to the web UI~

https://github.com/gutris1/sd-hub/blob/master/README.md#downloader

**Comment by NUROISEA** [comment]
I added it, but I cannot test since I currently do not have any colab credits

Can you test if the extension works?

**Comment by Ayaya70** [comment]
Tested. It works perfectly. Thank you very much!
